src/pages/home/home.tsx

```diff
@@ -1,15 +1,75 @@
 import TopProducts from '../../components/Product/TopProducts';
 import HomeCards from './components/IntroCards/HomeCards';
 import HomeParallax from './components/parallax/homeParallax';
+import ProductMountainSlider from './components/productMountainSlider/productMountainSlider';
 import styles from './home.module.scss';
+import useFetch from '../../hooks/useFetch';
+import { BACKEND_URL } from '../../utils/constants';
+import ProductCard from '../../components/Product/ProductCard';
+
+export interface cardDetails {
+  products: {
+    productImages: { url: string }[];
+    details?: string;
+    id: string;
+    category: string;
+    ratings?: number;
+    price: number;
+    bonus: number;
+    collectionId: string;
+    createdAt?: string;
+    expDate: string;
+    expiredflag: boolean;
+    name: string;
+    quantity: number;
+  }[];
+  message: string;
+  page: number;
+  totalPages: number;
+}
 
 function Home() {
+  const url = `${BACKEND_URL}/products/all?page=1&limit=10`;
+  const { data } = useFetch(url);
+  const instantData: cardDetails | null = data as cardDetails | null;
+  const products = instantData?.products;
+
   return (
     <div className={styles.home}>
-      <div className="homeSlides" data-testid="home">
-        <HomeParallax />
-        <HomeCards />
-        <TopProducts />
+      <div className="wrapperHome">
+        <div className="homeSlides" data-testid="home">
+          <HomeParallax />
+        </div>
+        <div className="newProducts">
+          <div className="title">New Release</div>
+          <div className="slides_of_Products">
+            {products && <ProductMountainSlider imageData={products} />}
+            <div className="over"></div>
+            <img src="./images/gif.gif" className="bg"></img>
```

Comment:
I honestly think this gif background doesn't look good. I fetched this branch and removed the background and it looks better. Consider removing it.